_includes/layouts/example.html

```diff
@@ -8,7 +8,7 @@ <h1>{{ title }}</h1>
   <time datetime="{{ page.date | htmlDateString }}">{{ page.date | readableDate }}</time>
   {%- if author %}
   <span class="author">
-    by&nbsp;<span class="author">{{ author }}</span>
+    by&nbsp;<span class="author"><a href="/contributors/{{ author }}">{{ author }}</a></span>
```

Comment:
Please use the `url` filter for propper URL generation, like here for example:
https://github.com/tbreuss/mithril-by-examples/blob/main/_includes/layouts/example.html#L31